### PR TITLE
update examples

### PR DESCRIPTION
**Pull request by forefab**

### CODE REVIEW

File: docs/foresight/metrics/groundedness.md [hunk 1]
```diff
@@ -3,19 +3,30 @@ sidebar_position: 1
...
+- **Response 1**: *7945.* `[groundedness score = 1.0]`
+- **Response 2**: *0000.* `[groundedness score = 0.0]`
+- **Response 3**: *1234.* `[groundedness score = 0.0]`
+- **Response 4**: *The code has been changed due to security reasons.* `[groundedness score = 1.0]` (100% grounded, even if not answering the question)
+
+Example:
+- **Context**: *Albert Einstein, (14 March 1879 - 18 April 1955) was a German-born theoretical physicist. In 1905, sometimes described as his annus mirabilis (miracle year), Einstein published four groundbreaking papers.*
+- **Question**: *How old did Einstein get and at what age did he publish four groundbreaking papers?*
+- **Response 1**: *He was 76 years old when he died and he published the four groundbreaking papers in the year he turned 26.* `[groundedness score = 1.0]`
+- **Response 2**: *He was 50 years old when he died and he published the four groundbreaking papers in the year he turned 26.* `[groundedness score = 0.5]` (Age is wrong, publish fact is correct)
```
Comment (miivanov90) [inline docs/foresight/metrics/groundedness.md:31]: FYI we score this as 0 today ;) You should probably pick an example where we give the same score :P

fact 01": {
                        "fact": "Einstein published four groundbreaking papers at the age of 26.",
                        "groundedness_score": 0.0,
                        "per_context_scores": [
                            0.0
                        ]
                    }

**Review by miivanov90 — APPROVED** [review]

File: docs/foresight/metrics/groundedness.md (hunk 1 above)
Comment (forefab) [reply docs/foresight/metrics/groundedness.md:31]: 1.0
{'root': {'compute': {'identifier': 'GroundMasala_20240421',
   'extract_facts': '',
   'compute_fact_groundedness': '',
   'facts': {'fact 00': {'fact': 'Einstein was more than 60 years old when he died.',
     'groundedness_score': 1.0,
     'per_context_scores': [1.0]},
    'fact 01': {'fact': 'Einstein published four groundbreaking papers in his twenties.',
     'groundedness_score': 1.0,
     'per_context_scores': [1.0]}}}}}

Comment (miivanov90) [reply docs/foresight/metrics/groundedness.md:31]: Yeah that's not the prod one - I think it's @gakhromov new NLI prompt - though I thought that one had worse metaeval scores.

Comment (miivanov90) [reply docs/foresight/metrics/groundedness.md:31]: ja genau, prod is 20240408 (the debug info I shared above was from the prod playground)

Comment (forefab) [reply docs/foresight/metrics/groundedness.md:31]: OK, found a new example where our prod score works as expected.

**Pull request merged**